src/sentry/relay/config/__init__.py

```diff
@@ -497,6 +497,7 @@ def _filter_option_to_config_setting(flt: _FilterSpec, setting: str) -> Mapping[
     [
         "s:transactions/user@none",
         "d:transactions/duration@millisecond",
+        "c:transactions/count_per_root_project@none",
```

Comment:
I think I added this line with @jjbayer and @untitaker, during a debugging session, otherwise, relay doesn't expose this new metric to clickhouse trough kafka.

Reply:
As of https://github.com/getsentry/relay/pull/1484, Relay does not require an allow list for transaction metrics anymore, so adding it here is not required AFAICT.

It might be that the `test_all_transaction_metrics_emitted` failed without the allowlist, but we disabled that test now.

Reply:
In that case it would be better to remove this here.

Reply:
done in 38888b401f3eb882be70e40ad47d25859a36596a